### PR TITLE
[11.0][MIG] l10n it reverse charge: Migration to 11.0

### PR DESCRIPTION
**Pull request by jackjack82**

### CODE REVIEW

**Comment by jackjack82** [comment]
@eLBati metodo "_get_company_partner" rinominato in "_compute_company_partner" per verifiche LINT. il campo è calcolato non storato, non dovrebbe dare problemi. Modifica in models.account_invoice.py righe 45 e 53

**Comment by eLBati** [comment]
@jackjack82 controlla travis plis

**Comment by jackjack82** [comment]
@eLBati travis ok. why is cl/runbot still giving error "2018-07-06 10:56:03,562 166 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view res.partner.pec.kanban (): use the hasclass(*classes) function to filter elements by their classes" if https://github.com/OCA/l10n-italy/pull/533/files is fine?

**Comment by eLBati** [comment]
@jackjack82 I see commits that should not be here, like https://github.com/OCA/l10n-italy/pull/531/commits/84fa8b3be58555309754008c58eaf951a1a99bea and https://github.com/OCA/l10n-italy/pull/531/commits/3c54d88a66fbe508a9780a466c899fbeb136b8ad
as already contained in ```11.0```
Did you add them to your branch instead of rebasing on top of ```11.0```?

File: l10n_it_reverse_charge/__manifest__.py [hunk 1]
```diff
@@ -7,14 +6,14 @@
...
-    'version': '10.0.1.1.1',
+    'version': '11.0.1.0.0',
     'category': 'Localization/Italy',
     'summary': 'Reverse Charge for Italy',
     'author': 'Odoo Italia Network,Odoo Community Association (OCA)',
     'license': 'LGPL-3',
     'website': 'https://www.odoo-italia.net',
     'depends': [
-        'account_accountant',
+        # 'account_menu_group',
```
Comment (SimoRubi) [inline l10n_it_reverse_charge/__manifest__.py]: Why commenting a dependency? Keep it or remove it

File: l10n_it_reverse_charge/models/account_invoice.py [hunk 2]
```diff
@@ -35,22 +34,43 @@ class AccountInvoice(models.Model):
         comodel_name='account.invoice',
         string='RC Self Purchase Invoice', copy=False, readonly=True)
 
+    rc_partner_supplier_id = fields.Many2one(
```
Comment (SimoRubi) [inline l10n_it_reverse_charge/models/account_invoice.py]: This is new functionality, please implement new functionalities out of the migration commit, thanks

File: l10n_it_reverse_charge/README.rst [hunk 3]
```diff
@@ -119,7 +119,8 @@ Contributors
 
 * Davide Corio
 * Alex Comba <alex.comba@agilebg.com>
-* Lorenzo Battistini <lorenzo.battistini@agilebg.com
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
```
Comment (SimoRubi) [inline l10n_it_reverse_charge/README.rst]: You can autogenerate a much nicer README using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)

**Comment by eLBati** [comment]
@jackjack82 please consider including https://github.com/OCA/l10n-italy/pull/559

**Comment by alessandrocamilli** [comment]
@eLBati possiamo passare a https://github.com/OCA/l10n-italy/pull/658 ? e chiudere questa PR?

**Comment by eLBati** [comment]
:+1:

**Pull request closed without merging**